proposed/security-disclosure-meta.md

```diff
@@ -65,6 +65,29 @@ outside authority for something as sensitive as security related topics.
 However due to defined locations and formats, it will become possible for
 other people to build centralized tools around this information.
 
+For further reference here is a list of security disclosure processes in
+various PHP and non-PHP projects:
+
+* http://symfony.com/doc/current/contributing/code/security.html
+* http://framework.zend.com/security/
+* http://www.yiiframework.com/security/
+* https://www.drupal.org/security
+* http://codex.wordpress.org/FAQ_Security
+* http://www.sugarcrm.com/page/sugarcrm-security-policy/en
+* http://typo3.org/teams/security/
+* http://cakephp.org/development
+* http://www.concrete5.org/developers/security/
+* http://developer.joomla.org/security.html
+* http://wiki.horde.org/SecurityManagement
+* http://www.revive-adserver.com/support/bugs/
```

Comment:
http://www.revive-adserver.com/security/ is the main one, although we also repeat the same text in the bugs page.